.github/filters.yml

```diff
@@ -1,3 +1,5 @@
+mgmt:
+  - .github/workflows/**
```

Comment:
Could you add a top comment on how this file is used? I keep wondering how it fits into the labeling process.